Fix JavaScript lint errors

## JavaScript Linting Failures

  Linting failures were detected in the automated JavaScript lint workflow run.

  ### Workflow Details

  - Run: https://github.com/stdlib-js/stdlib/actions/runs/15313072346
  - Type: JavaScript Linting
  - Date: 2025-05-29 00:11:49 UTC

  ### Error Details
  ```
  make[1]: Entering directory '/home/runner/work/stdlib/stdlib'

Linting file: lib/node_modules/@stdlib/math/strided/special/srsqrt/lib/ndarray.js

Linting file: lib/node_modules/@stdlib/blas/base/wasm/saxpy/lib/binary.js

Linting file: lib/node_modules/@stdlib/stats/iter/mprod/lib/main.js

Linting file: lib/node_modules/@stdlib/blas/ext/base/sapx/lib/sapx.js

Linting file: lib/node_modules/@stdlib/ndarray/base/includes/lib/10d_complex.js

Linting file: lib/node_modules/@stdlib/assert/is-float32matrix-like/lib/index.js

Linting file: lib/node_modules/@stdlib/ndarray/map/lib/index.js

Linting file: lib/node_modules/@stdlib/stats/base/mskrange/lib/main.js

Linting file: lib/node_modules/@stdlib/math/base/special/floorf/lib/index.js

Linting file: lib/node_modules/@stdlib/stats/strided/dnanmean/lib/dnanmean.native.js

Linting file: lib/node_modules/@stdlib/time/minutes-in-year/lib/index.js

Linting file: lib/node_modules/@stdlib/stats/base/dists/frechet/mode/lib/main.js

/home/runner/work/stdlib/stdlib/lib/node_modules/@stdlib/stats/base/dists/frechet/mode/lib/main.js
  30:28  warning  Unknown word: "échet"  @cspell/spellchecker

✖ 1 problem (0 errors, 1 warning)


Linting file: lib/node_modules/@stdlib/blas/ext/base/snansum/lib/ndarray.native.js

Linting file: lib/node_modules/@stdlib/constants/int16/max/lib/index.js

Linting file: lib/node_modules/@stdlib/ndarray/base/unary-by/lib/9d_blocked.js

Linting file: lib/node_modules/@stdlib/assert/is-complex64/lib/main.js

Linting file: lib/node_modules/@stdlib/complex/float64/ctor/lib/index.js

Linting file: lib/node_modules/@stdlib/ndarray/base/unary-reduce-subarray/lib/5d_blocked_accessors.js

Linting file: lib/node_modules/@stdlib/math/base/special/csch/lib/native.js

Linting file: lib/node_modules/@stdlib/_tools/eslint/rules/jsdoc-table-pipes/lib/main.js

Linting file: lib/node_modules/@stdlib/math/iter/special/cos/lib/index.js

Linting file: lib/node_modules/@stdlib/napi/argv-strided-float32array/lib/browser.js

Linting file: lib/node_modules/@stdlib/array/base/without/lib/main.js

Linting file: lib/node_modules/@stdlib/math/strided/special/floor/lib/index.js

Linting file: lib/node_modules/@stdlib/stats/strided/svariancepn/lib/svariancepn.native.js

Linting file: lib/node_modules/@stdlib/stats/base/dists/beta/kurtosis/lib/native.js
make[1]: Leaving directory '/home/runner/work/stdlib/stdlib'
  ```

  ### Pull Request Instructions

  -   Please use the following PR title format:
  "chore: fix JavaScript lint errors (issue #<ISSUE_NUMBER>)".
  -   Reference this issue in the "Related Issues" section of the PR body as "resolves #<ISSUE_NUMBER>".

--- a/etc/eslint/rules/spellcheck.js
+++ b/etc/eslint/rules/spellcheck.js
@@ -115,7 +115,8 @@ rules[ '@cspell/spellchecker' ] = [ 'warn', {
 			'tricube',
 			'trigamma',
 			'uncapitalize',
-			'unregularized'
+			'unregularized',
+			'Fréchet'
 		]
 	}
 }];

